arangodb-net-standard/DocumentApi/PatchDocumentResult.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ArangoDBNetStandard.DocumentApi
+{
+    public class PatchDocumentResult
```

Comment:
We also need to handle `Old` and `New` as we do with `PostDocumentResponse`

see documentation:

> returnOld (optional): Return additionally the complete previous revision of the changed document under the  attribute old in the result.
> 
> returnNew (optional): Return additionally the complete new document under the attribute new in the result.

Reply:
Actually... since I think we'd want to use template parameter in the method to define the type of the returned object, we could perhaps create a separate overloaded method `PatchDocumentAsync<T>` for when you care about the type of `New` or `Old`.

Reply:
I've since left another comment on the PatchDocumentasync method itself.  I'm wondering if we need to allow a user to define types for both the update model as well as the possibly-returned New or Old document model. Let's keep further comments on this to the other thread on PatchDocumentAsync.